app/models/miq_widget_set.rb

```diff
@@ -17,6 +17,11 @@ class MiqWidgetSet < ApplicationRecord
 
   validates :description, :uniqueness_when_changed => {:scope   => [:owner_id, :userid],
                                                        :message => _("must be unique for this group and userid")}
+
+  # group used to be for only group widgets, now it is for all widgets
```

Comment:
Can you cite where this happened, or how you know this?

Reply:
yea, I remember when it happened in libor's changes. will track down